keep/api/models/db/provider.py

```diff
@@ -7,7 +7,7 @@
 class Provider(SQLModel, table=True):
     id: str = Field(default=None, primary_key=True)
     tenant_id: str = Field(foreign_key="tenant.id")
-    name: str
+    name: str = Field(unique=True)
```

Comment:
@pehlicd this could be problematic because we need it to be unique per tenant id.

Please see `db/preset.py` (__table_args__ = (UniqueConstraint("tenant_id", "name"),))

Reply:
@talboren I added table args do you think I should remove `Field(unique=True)` this one. In db/preset.py name is like that as well so I left this one

Reply:
you can remove it :)